### PR TITLE
Add database configuration variables.

### PR DESCRIPTION
**Pull request by naoitoi**

### CODE REVIEW

File: lib/config.js [hunk 1]
```diff
@@ -153,9 +153,18 @@ function parseDatabaseConfig (prefix) {
   // When using SQLite in-memory database, default to sync enabled
   const sync = castBool(getEnv(prefix, 'DB_SYNC'), uri === 'sqlite://')
 
+  // Knex
+  const knex_env = getEnv(prefix, 'DB_KNEX_ENV') || 'development'
+  // Oracle
```
Comment (lumberj) [inline lib/config.js]: Is this necessarily Oracle specific?

File: lib/config.js [hunk 2]
```diff
@@ -153,9 +153,18 @@ function parseDatabaseConfig (prefix) {
...
+  const connection_user = getEnv(prefix, 'DB_CONNECTION_USER')
```
Comment (lumberj) [inline lib/config.js]: I think we're using camelCase for everything aside from JSON responses

File: lib/config.js [hunk 3]
```diff
@@ -153,9 +153,18 @@ function parseDatabaseConfig (prefix) {
...
+  const connection_password = getEnv(prefix, 'DB_CONNECTION_PASSWORD')
```
Comment (lumberj) [inline lib/config.js]: Can you add unit tests for these two options?

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "five-bells-shared",
-  "version": "11.2.1",
+  "version": "11.2.2",
```
Comment (lumberj) [inline package.json]: Version bumping should not be part of a PR as we discussed earlier

Comment (lumberj) [inline package.json]: Use `npm run bump` after merging changes

File: lib/config.js (hunk 3 above)
Comment (naoitoi) [inline lib/config.js]: This line does exactly the same thing as other getEnv() lines in this file, and this code will be exercised by ilp-server integration test.  Therefore, I don't think unit test is necessary.

File: lib/config.js (hunk 2 above)
Comment (naoitoi) [inline lib/config.js]: Please see: https://github.com/interledger/five-bells-shared/blob/master/lib/config.js#L109

Comment (lumberj) [inline lib/config.js]: :+1:

File: lib/config.js [hunk 5]
```diff
@@ -153,9 +153,19 @@ function parseDatabaseConfig (prefix) {
   // When using SQLite in-memory database, default to sync enabled
   const sync = castBool(getEnv(prefix, 'DB_SYNC'), uri === 'sqlite://')
 
+  // Knex
+  const knex_env = getEnv(prefix, 'DB_KNEX_ENV') || 'development'
```
Comment (lumberj) [inline lib/config.js]: Shouldn't DB_KNEX_ENV be `test` in a test environment?

**Comment by lumberj** [comment]
LGTM

**Comment by naoitoi** [comment]
This pointed to a wrong merge target, so I will close this PR. 
I have created new PR, 106, and merged that one. 
BOth PR105 and PR106 have the exact same changes.

**Pull request closed without merging**